src/android/com/outsystems/plugins/healthfitness/oshealthfitnesslibrary/background/VariableUpdateService.kt

```diff
@@ -51,6 +51,7 @@ class VariableUpdateService : BroadcastReceiver() {
 
         val db = DatabaseManager.getInstance(context)
         val backgroundJobs = db.fetchBackgroundJobs(variableName)
+        val jobs = db.fetchBackgroundJobs()
```

Comment:
We need to remove this. I added just for testing 😅

Reply:
And forgot to remove

Reply:
Done 😄